.github/CONTRIBUTING.md

```diff
@@ -76,12 +76,18 @@ Following a small guidance how we suggest to start.
    We suggest [IntelliJ Community Edition](https://www.jetbrains.com/idea/download/)
    and install the following plugins:
    - EditorConfig by JetBrains (might be already activated)
-   - Spek Framework by Spek Team 
+   - Spek by Ranie Jade Ramiso
```

Comment:
That's the plugin for spek 1.x and usually new contributors should only use spek 2.x 
Thus I am not sure if it makes sense to mention it as well but if you think yes, then please add both as follows:
```suggestion
   - Spek Framework by Spek Team (for Spek 2.x)
   - Spek by Ranie Jade Ramiso (for Spek 1.x)
```

Reply:
Strangely I couldn't find Spek Framework in the plugin market when I originally searched for it, but I see it now. So I think we should leave it as it was before?

Reply:
Most likely because spek wasn't yet (or still isn't) available for intellij 2019.3